### PR TITLE
Moved files to root

### LINKED ISSUE #3: Move the project to the root of the repository
The content of the following folder should be in the root of the repository: https://github.com/benchflow/spark-tasks-sender/tree/dev/spark-task-sender

### LINKED ISSUE #4: The package should correspond to benchflow.cloud
See https://github.com/benchflow/client

### PR DESCRIPTION
**Pull request by Cerfoglg**
Fixes #3 and fixes #4

### CODE REVIEW

File: .project [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <projectDescription>
-	<name>benchflow-spark-tasks-sender</name>
+	<name>Spark-tasks-sender</name>
```
Comment (VincenzoFerme) [inline .project]: Why did you change the project name?

Comment (Cerfoglg) [inline .project]: Most of the others didn't have benchflow in their name. I don't think you need to specify its for benchflow, since it's already in the benchflow repo anyways.

**Comment by Cerfoglg** [comment]
@VincenzoFerme Should be read to merge now

File: .project (hunk 1 above)
Comment (VincenzoFerme) [inline .project]: In most of the projects in the benchflow repo we are using the benchflow- prefix in the name. This is because when you load the project in an IDE, you get the reference to the BenchFlow framework as part of the project name.

**Comment by VincenzoFerme** [comment]
@Cerfoglg I don't see the following issue solved: https://github.com/benchflow/spark-tasks-sender/pull/8#discussion_r46414714

**Comment by Cerfoglg** [comment]
@VincenzoFerme I renamed it. I'll also make sure to rename other projects in case they are missing benchflow in the title
